Scala/Scala.sublime-syntax

```diff
@@ -138,7 +138,7 @@ contexts:
           pop: true
         - match: '([a-zA-Z$_][a-zA-Z0-9$_]*)\s*:\s*'
           captures:
-            1: entity.name.variable variable.parameter
+            1: entity.name.parameter variable.parameter
```

Comment:
This is one of those weird interactions between my PRs.  Once all the PRs land, I'll go back and make sure that the interaction actually worked out.

Scala does not make a distinction between entities that are parameters and entities that are objects or variables, and the distinction between these two and entities that are functions is microscopic.  For that reason, I would rather actually use `entity.name.parameter`.  This is more strongly seen in the pattern matching PR.  `variable.parameter` is fine, but it still needs to be considered by symbol indexing.